CHANGELOG.md

```diff
@@ -5,7 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 
-## Ivanchuk-Beta2
+## Unreleased as of Sprint 118 - 2019-08-19
+
+### Added
+- Append RBAC features for VMRC console request through the API [(#642)](https://github.com/ManageIQ/manageiq-api/pull/642)
```

Comment:
Something got mixed up...?? This is a PR for manageiq-api.

Reply:
oopsy,..my bad. Closing this PR.